OpenRA.Mods.Common/Orders/EnterTransportsTargeter.cs

```diff
@@ -20,7 +20,7 @@ public class EnterTransportsTargeter : EnterAlliedActorTargeter<CargoInfo>
 		readonly AlternateTransportsMode mode;
 
 		public EnterTransportsTargeter(string order, int priority,
```

Comment:
The current version of #16607 also removes this completely.

Reply:
Removed in rebase.